db/models.js

```diff
@@ -93,10 +93,46 @@ function fetchPostCommentsByArticleId(articleId, author, body) {
     *;`;
     return db.query(sqlQuery, [articleId, author, body]).then((response) => {
         if (response.rows.length === 0) {
-            return Promise.reject({status: 404, error: "Invalid query"});
+            return Promise.reject({ status: 404, error: "Not found" });
+        }
+        return response;
+    })
+}
+
+function fetchPatchArticleByArticleId(articleId, incVotes) {
+    if (articleId % 1 !== 0 || articleId <= 0) {
```

Comment:
This is a little unnecessary, but up to you if you wanted to keep it. If you did want to include a clause for none whole numbers then you should test for it too.